backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/StartVmBackupCommand.java

```diff
@@ -516,7 +516,7 @@ private Guid createVmBackup() {
         return vmBackup.getId();
     }
 
-    private void setDiskBackupModeIfSupported(DiskImage disk) {
+    protected void setDiskBackupModeIfSupported(DiskImage disk) {
```

Comment:
Please avoid using this outside of StartVMbackupCommand - it is likely to be wrong in context of hybrid backup.

If you need to call this copy the code to hybrid backup. If you are sure the code is
correct and it should be shared move it to another class used by both legacy and hybrid backup.